scala-package/infer/pom.xml

```diff
@@ -101,5 +101,10 @@
             <version>1.10.19</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
```

Comment:
I am guessing you added this to get Pair, If you make the return type a POJO, you won't need this.